shotgun_api3/shotgun.py

```diff
@@ -2587,7 +2587,10 @@ def _upload_to_sg(self, entity_type, entity_id, path, field_name, display_name,
             url = urllib.parse.urlunparse((self.config.scheme, self.config.server,
                                            "/upload/upload_file", None, None, None))
             if display_name is None:
-                display_name = os.path.basename(path)
+                if six.PY3:
```

Comment:
Minor comment: I'd prefer having the opposite test: if Py2:... else.
So the specific is about Py2.